solr/modules/llm/src/test/org/apache/solr/llm/embedding/DummyEmbeddingModel.java

```diff
@@ -0,0 +1,61 @@
+/*
+ * Licensed to the Apache Software Foundation (ASF) under one or more
+ * contributor license agreements.  See the NOTICE file distributed with
+ * this work for additional information regarding copyright ownership.
+ * The ASF licenses this file to You under the Apache License, Version 2.0
+ * (the "License"); you may not use this file except in compliance with
+ * the License.  You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package org.apache.solr.llm.embedding;
+
+import dev.langchain4j.data.embedding.Embedding;
+import dev.langchain4j.data.segment.TextSegment;
+import dev.langchain4j.model.embedding.EmbeddingModel;
+import dev.langchain4j.model.output.Response;
+import java.util.List;
+
+public class DummyEmbeddingModel implements EmbeddingModel {
```

Comment:
1/2 Maybe we could make this dummy model a little more 'realistic' by giving it parameters e.g. like https://github.com/SeaseLtd/solr/pull/5/commits/465383748b0f35847cbebbd024bd082cc8e23ab5 for example.

Reply:
Done, I got inspired from your commit and added an array parameter, feel free to add additional considerations where needed!